src/metaswitch/clearwater/etcd_tests/etcdtestbase.py

```diff
@@ -35,9 +35,10 @@
 import sys
 import unittest
 from .etcdcluster import EtcdCluster
+from metaswitch.common.logging_config import configure_test_logging
+
```

Comment:
for consistency, can you match the layout of these two lines between this file and etcdcluster.py, above.
i.e. separate both the import and configure as a single block, or separate the configure from the whole import block.